e2e/pages/app/app.module.ts

```diff
@@ -122,17 +143,32 @@ const ROUTES: Routes = [
     path: 'select',
     component: SelectTestPageComponent
   }, {
+    path: 'side-inset-panel-splitter',
+    component: SideInsetPanelSplitterNg1TestPageComponent
+  }, {
     path: 'sliders',
     component: SlidersTestPageComponent
   }, {
+    path: 'split-button-dropdowns',
+    component: SplitButtonDropdownsTestPageComponent
+  }, {
+    path: 'stacked-tabs',
+    component: StackedTabsNg1TestPageComponent
+  }, {
+    path: 'tabs',
+    component: TabsNg1TestPageComponent
+  }, {
     path: 'tags',
     component: TagsTestPageComponent
   }, {
     path: 'toggle-button',
     component: ToggleButtonTestPageComponent
   }, {
-    path: 'toggleswitches',
-    component: ToggleSwitchesTestPageComponent
+    path: 'toggle-button',
```

Comment:
This route appears to be duplicated - i think this should be toggle switches instead